packages/nx-aws-cache/src/tasks-runner/encryptor/encrypt.ts

```diff
@@ -0,0 +1,45 @@
+import { Transform, TransformCallback } from 'stream';
```

Comment:
I don't know much about this logic. Is this a standard way of encrypting content? What I recently used is this library: `@aws-crypto/client-node`. Have you checked on this?

Reply:
@aws-crypto/client-node Use rsa and AWS kms is not compatible with 32 characters key long. https://nx.dev/ci/recipes/security/encryption